src/components/Common/MainMenu.vue

```diff
@@ -35,6 +35,16 @@
         >
           Security
         </b-nav-item>
+        <b-nav-text
+          class="ml-2"
+          title="Give us your feedback on our GitHub page"
+        >
+          <a
+            href="https://github.com/kuzzleio/kuzzle-admin-console"
```

Comment:
Maybe we should redirect to the repository issues section
```suggestion
            href="https://github.com/kuzzleio/kuzzle-admin-console/issues/new"
```